.env.test

```diff
@@ -13,4 +13,4 @@ GOOGLE_TAG_MANAGER_APPLY=GTM-APPLY-XX
 GOOGLE_TAG_MANAGER_MANAGE=GTM-MANAGE-XX
 BULLET=false
 COVERAGE=false
-ZENDESK_MESSAGING_KEY=bc5a052c-d446-45f6-b2d7-b6a90de7e250
+ZENDESK_MESSAGING_KEY=9b56d751-c185-4ccd-947b-e7d603b236fe
```

Comment:
is this a real key 🙈

Reply:
Yes. It's a sandbox key. We need a real key to open the widget.

Reply:
I'm concerned that this is in our source code and not hidden anywhere 🤔

Reply:
This is now a github secret